app/components/emoji_picker/emoji_picker.js

```diff
@@ -43,10 +43,19 @@ class EmojiPicker extends PureComponent {
 
         this.state = {
             emojis: props.emojis,
-            searchTerm: ''
+            searchTerm: '',
+            deviceWidth: Dimensions.get('window').width - (SECTION_MARGIN * 2)
```

Comment:
Why move the margin into the state? Since it's no longer the width of the screen, maybe it should be called just width to make it clearer to anyone who expects it to be the width of the whole thing